app/views/home/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="main">
```

Comment:
# [OPTIONAL]
- You are supposed to follow the design guidelines and make sure you follow:
  * Colors
  * Typography: font face, size and weight.
  * Layout: composition and space between elements.
 Here, you made the buttons red and they should be placed towards the bottom of the page

### Your Implementation
![image](https://github.com/shihabrafio/Budget_App/assets/45460192/e22126c2-e65e-4e09-a2c9-2bd4fe68d402)
### Design guidelines
![image](https://github.com/shihabrafio/Budget_App/assets/45460192/41144cda-f549-4745-a04a-741b2f26ab0c)